libi2pd/NetDb.cpp

```diff
@@ -922,6 +922,10 @@ namespace data
 		else if(!m_FloodfillBootstrap)
 			LogPrint (eLogWarning, "NetDb: Requested destination for ", key, " not found");
 
+		// All peers hashs in buffer?
+		if(msg->GetPayloadLength() < (size_t) (33 + num * 32))
```

Comment:
Use constant instead 32.

Reply:
What's the constant name for a router identity hash? I don't see anything already defined in any headers.

I just copied the values from with-in the for() loop below it.
```
                for (int i = 0; i < num; i++)
                {
                        const uint8_t * router = buf + 33 + i*32;
```

Reply:
If no suggestions are given, will add new IDENTITY_HASH_SIZE constant to Identity.h.